rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMPage.java

```diff
@@ -438,11 +439,14 @@ public void onClick(ClickEvent event) {
         schedulerLinkButton.setIcon(Images.instance.scheduler_30().getSafeUri().asString());
         schedulerLinkButton.setIconSize(25);
         schedulerLinkButton.setBorder(GREY_BUTTON_BORDER);
-	ToolStripButton cloudAutomationLinkButton = getCloudAutomationLinkButton();
-         cloudAutomationLinkButton.setIcon(Images.instance.pca_30().getSafeUri().asString());
-	 cloudAutomationLinkButton.setIconSize(25);
-	 cloudAutomationLinkButton.setBorder(GREY_BUTTON_BORDER);
-
+	    ToolStripButton cloudAutomationLinkButton = getCloudAutomationLinkButton();
+        cloudAutomationLinkButton.setIcon(Images.instance.pca_30().getSafeUri().asString());
+	    cloudAutomationLinkButton.setIconSize(25);
+	    cloudAutomationLinkButton.setBorder(GREY_BUTTON_BORDER);
+        ToolStripButton notificationPortalLinkButton = getNotificationPortalLinkButton();
```

Comment:
As you mentioned during the Standup, this code could be factorized, at least in the method here.